draft/accessibility-team.rst

```diff
@@ -0,0 +1,158 @@
+
+To aid in reviewing, the Accessibility Team members shall be added to a team
+in the GitHub organization with read access to relevant repositories, so that
+they may be requested to review pull requests, at the discretion of the author,
+reviewer, or other party.
+
+Many of these duties can be undertaken by any contributor, not only by the
+Accessibility Team, however the Accessibility Team exists to coordinate this
+work and to step in where contributors are not availble and support those who
+lack the knowledge to do so themselves.
+
+Motivation
+==========
+
+Accessible websites are not technically difficult to create, but accessibility
+is often overlooked when developing features. Django is no exception to this,
+and given Django's commitment to diversity and inclusion, the lack of
+accessibility in many parts of Django falls short of an acceptable standard.
+
+Improving accessibility also has a tendency to improve the experience of other
+users. This is called the
+`curb cut effect <https://alexwlchan.net/2019/01/monki-gras-the-curb-cut-effect/>`_
+While this is mostly noted in the real world, the web also experiences this
+affect. For example, keyboard shortcuts are used to allow keyboard-only users
+navigate a site more efficiently, but this positive benefit affects everyone,
+not just the physically disabled.
+
+The django admin currently exhibits many accessibility issues, such as forms
+fields missing labels, low color contrast, a lacklustre keyboard-only
+experience among other issues. Other parts of the project have problems such
+as non-semantic HTML generated by forms. Futher problems exist on other sites
+maintained by the DSF, such as the
+`Django website <https://www.djangoproject.com/>`__,
+`Django Snippets <https://djangosnippets.org/>`_, and others.
+
+Rationale
+=========
+
+An alternative is to go on as usual, leaving accessibility in the hands of
+individual contributors when writing and reviewing code. However, given the
+accessibility issues that exist in the Djago admin and on the
+`Django website <https://www.djangoproject.com/>`__, this doesn't seem to be
+enough.
+
+Another option is to implement some basic standards, such as conforming to WCAG
+and setting up a CI to test. This approach is better than nothing but it
+lacks a clear process for deciding on these. Ongoing maintenence would also be
+necessary to keep any CI or other tooling up to date with Django's code, along
+with manual audits - as
+`automatic processes cannot find every issue <https://alphagov.github.io/accessibility-tool-audit/>`_
+- and these could be easily forgotten.
+
+Resources
+=========
+
+- `Diverse Abilities and Barriers (W3C)
+  <https://www.w3.org/WAI/people-use-web/abilities-barriers/>`_
+- `Accessibility, Usability, and Inclusion (W3C)
+  <https://www.w3.org/WAI/fundamentals/accessibility-usability-inclusion/>`_
+- `Web Content Accessibility Guidelines (WCAG) Overview
+  <https://www.w3.org/WAI/standards-guidelines/wcag/>`_
```

Comment:
Could be a good place to mention [ATAG](https://www.w3.org/TR/ATAG20/). Its Part A is "authoring tools should be accessible", which is essentially what we’re discussing here for the Django admin. Its Part B isn’t something we’ve discussed yet, but I’d love to work on that as well if we agreed it was valuable.